### PR TITLE
Make AIM width customizable

### LINKED ISSUE #35516: can we have advanced inventory window wider?
can we have wider inventory window? or at least have an option to set its width?
long item names don't fit, but we have a lot of useless space on each side.

изображение

### PR DESCRIPTION
**Pull request by Night-Pryanik**
#### Summary
```SUMMARY: Interface "Make AIM window width customizable."```

#### Purpose of change
Closes #35516 - can we have advanced inventory window wider?

#### Describe the solution
Added an true-false option for width of the AIM window. It allows either occupying the full screen or leaving sidebar visible.

#### Describe alternatives you've considered
Allow setting a value of user's choice for AIM width.

#### Testing
Changed setting on and off, changed window size to really small and really big, looked if all is ok.

#### Additional context
Full screen on high resolution
изображение

Sidebar visible on high resolution
изображение

Full screen on low resolution
изображение

Sidebar visible on low resolution
изображение

### CODE REVIEW

File: src/options.cpp [hunk 1]
```diff
@@ -1491,6 +1491,11 @@ void options_manager::add_options_interface()
     "Vertical"
        );
 
+    add( "AIM_WIDTH", "interface", translate_marker( "Full screen Advanced Inventory Manager" ),
+        translate_marker( "Should Advanced Inventory Manager fit full screen, or leave sidebar visible." ),
```
Comment (OzoneH3) [inline src/options.cpp]: This text with a true/false option is not very clear on what will happen, maybe use strings fullscreen/sidebar visible or something?

Comment (Night-Pryanik) [reply src/options.cpp]: Rewrote description a bit. Hope it's more clear now.

Comment (Night-Pryanik) [reply src/options.cpp]: Rewrote description a bit. Hope it's more clear now.

**Comment by SirPendrak** [comment]
SUMMARY: Interface "Adds new mutation category 'Mouse'"

**Comment by Night-Pryanik** [comment]
Fixed, thank you.

**Pull request merged**